### ISSUE TITLE
I have tried to send spoofing emails but i have some problems since 2 months and i can't resolve them. I saw a lot of issues and nobody have a solution to my problem, also i ask in the irc channel but no response is the only i got.  This is the full output: [!] It appears your password was incorrect. Printing response: a bytes-like object is required, not 'str'    Press <return> to continue [*] SET has finished sending the emails    Press <return> to continue If someone can help me please, i tried all and i don't get result, please. Thank you.

### ISSUE THREAD

**Issue by devilbymind** [trustedsec/social-engineer-toolkit#939]
**Expected Behaviour**

Put here what the expected behaviour should be when reporting an issue

**Actual Behaviour**

Put here what the actual behaviour should be when reporting an issue

**Set Version**

_Set version_

Before opening an issue please make sure you are running the most up to date version of set

We see the same bug reports being opened for issues that have already been fixed in a new release

**Set in Kali**

Please make sure you are running kali-rolling and your system is up to date.
To update your system do the following ``apt update && apt full-upgrade``

Run the following to check your version of Kali ``lsb_release -rdc`` and you should see the following
output
```
Description:	Kali GNU/Linux Rolling
Release:	kali-rolling
Codename:	kali-rolling
````
If you do not see the following output download the ISO from http://cdimage.kali.org/kali-images/kali-weekly/

If you do all of the above and still have an issue then please open a issue.

**Comment by devilbymind** [comment]
please someone help me i cant solve this issue an i am not able to send any phishing emails due to this

**Comment by whynotmerijaan** [comment]
same

**Comment by Keynotey** [comment]
Did anyone find a solution to this?

**Comment by sanjanamalik1** [comment]
> 

I did all the above, updated as well but still showing the password was incorrect. Any solution possibly?

**Comment by BennyWestsyde** [comment]
There is a bug in smtp_web.py
it is on lines 463-465

Original:
        try:
            mailServer.docmd("AUTH LOGIN", base64.b64encode(provideruser.encode()))
            mailServer.docmd(base64.b64encode(pwd.encode()), "")

Updated:
        try:
            mailServer.docmd("AUTH LOGIN", base64.b64encode(provideruser.encode('utf-8')))
            mailServer.docmd(base64.b64encode(pwd.encode('utf-8')), b"")

_Originally posted by @BennyWestsyde in https://github.com/trustedsec/social-engineer-toolkit/issues/769#issuecomment-1182474137_

**Comment by clark1816** [comment]
> 

You need to use a google app password. I made a youtube video on how to fix the issue.  You have enable 2 step verification on you account and set up a app password and use the app password not your gmail password. https://youtu.be/ois43MH55Uc

**Comment by clark1816** [comment]
> Did anyone find a solution to this?

yes i fixed it here a video on the fix https://youtu.be/ois43MH55Uc

**Comment by HackingDave** [comment]
Closing as an unauthorized account/password or account-access request. SET is for authorized security testing only and this repository will not provide help for unauthorized access.
